### DISCUSSION TITLE
Next/Image not covering the parent div

### DISCUSSION THREAD

**Discussion by HenriqueRamos13** [vercel/next.js#25119] [Help]
I have this code: 

```
<div style={{width: "100%"}}>
      {infos?.imageUrl && (
        <Image
          src={infos.imageUrl}
          alt="draw"
          layout="fill"
          quality={55}
          objectFit="cover"
          objectPosition="center"
        />
        // <img
        //   src={infos.imageUrl}
        //   style={{ objectFit: "cover", width: "100%" }}
        // />
)}
```

Using the ```Image``` component of next, the image does not appear on the screen, but inspecting the chrome developer tools, I see that the parent div is with its 300px. However, if I use the code that is commented, using ```<img>``` tag, the image appears correctly. How can I solve this using next ```Image``` component?

I tried to use className attribute and put this code but nothing has changed.
```
<style jsx global>{`
        .nextJsImage {
          object-fit: cover;
        }
`}</style>
```

**Comment by bobarros** [comment]
Since you're choosing to work with fill, try an experiment.

Delete the image and replace it with something like:
```html
<p>experiment</p>
```

Does the paragraph takes place with the same sizes as what you're expecting the image to take? If not, you're looking at the problem in a wrong perspective.

**Reply by HenriqueRamos13** [reply]
But with the ```<img> ``` tag works. Is just with ```<Image>``` the problem. 
I tried with other elements, in the case ```object-fit``` is only for images i guess, so with other elements i set ```width: "100%"``` and works.

**Comment by HenriqueRamos13** [comment]
The ```object-fit``` when used with the ```<img>``` tag generates an ideal height for the image, this is what the Next ```<Image>``` is not doing. If I set a fixed height for the image's parent, it appears. However, the images have different sizes and are not fixed. Why the ```objectFit``` prop of ```<Image>``` not set a height?

**Comment by eddsaura** [comment]
Have you make sure the parent div is relative?

**Reply by HenriqueRamos13** [reply]
Yes. I tried with and without relative. In both cases dont work
